vector/src/main/java/im/vector/app/features/location/LocationTracker.kt

```diff
@@ -127,6 +128,19 @@ class LocationTracker @Inject constructor(
                 }
             }
         }
+
+        val nowMillis = System.currentTimeMillis()
+        val elapsedMillis = nowMillis - (lastLocationUpdateMillis ?: 0)
+        if (elapsedMillis >= MIN_TIME_TO_UPDATE_LOCATION_MILLIS) {
+            lastLocationUpdateMillis = nowMillis
+            notifyLocation(location)
```

Comment:
Nit You could use our handy `FirstThrottler`.

Reply:
Thanks for the info. In fact, I realized a debouce operation is more suitable here. I will try to rework on it.